client/examples/teradata/bteq/clean.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
```

Comment:
Ideally we would symlink these and the next few shell scripts to a common directory shared among all examples, but the README advocates that examples be copied out of the repo and into a folder elsewhere on the user's filesystem, which would break the symlinks. So I've simply copied them verbatim from the existing teradata/sql example.

Reply:
(fyi)